examples/00-mapdl-examples/basic_dpf_example.py

```diff
@@ -26,7 +26,9 @@
 from ansys.mapdl.core.examples import vmfiles
 
 ###############################################################################
-# ## Model creation
+# Model creation
+# ~~~~~~~~~~~~~~
+#
 # Running a verification manual example in MAPDL
```

Comment:
```suggestion
# Run a verification manual example in MAPDL
```
Not sure what is meant by manual--Should it be: Run a verification example manually in MAPDL

Reply:
I meant from the verification guide